src/Extension/Phpunit/DatabaseTestListener.php

```diff
@@ -0,0 +1,165 @@
+<?php
+
+/*
+ * This file is part of the Symfony CMF package.
+ *
+ * (c) 2011-2015 Symfony CMF
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+
+namespace Symfony\Cmf\Component\Testing\Extension\Phpunit;
+
+use Symfony\Cmf\Component\Testing\Exception\ManagerNotFoundException;
+use Symfony\Cmf\Component\Testing\Database\Manager\ManagerInterface;
+use Symfony\Cmf\Component\Testing\Database\Manager\OrmManager;
+use Symfony\Cmf\Component\Testing\Database\Manager\PhpcrManager;
+use Symfony\Cmf\Component\Testing\Exception\SetupFailedException;
+use Symfony\Cmf\Component\Testing\RequiresDatabaseInterface;
+use Symfony\Component\Process\ProcessBuilder;
+use Symfony\Component\Process\PhpExecutableFinder;
+use Doctrine\Common\DataFixtures\Purger;
+
+class DatabaseTestListener implements \PHPUnit_Framework_TestListener
+{
+    /**
+     * @var ProcessBuilder
+     */
+    private $processBuilder;
+    private $prefix = array();
+    private $initializedDatabases = array();
+    private $needsDrop = false;
+    private $currentDriver;
+    /**
+     * @var ManagerInterface[]
+     */
+    private $managers = array();
+
+    public function __construct($processBuilder = null, array $managers = array())
+    {
+        if (null === $processBuilder) {
+            $this->processBuilder = new ProcessBuilder();
+            $phpExecutableFinder = new PhpExecutableFinder();
+            $phpExecutable = $phpExecutableFinder->find(false);
+            if (false === $phpExecutable) {
+                throw new \RuntimeException('No PHP executable found on the current system.');
+            }
+
+            // Symfony 2.3 does not support array prefix, so we have to implement it ourselves
+            $this->prefix = array($phpExecutable, __DIR__.'/../../bin/console');
```

Comment:
what about the default Symfony structure putting the console in ``app/console`` ?